packages/superset-ui-plugin-chart-word-cloud/src/transformProps.ts

```diff
@@ -13,14 +13,14 @@ function transformData(data: ChartProps['queryData'][], formData: ChartProps['fo
 
 export default function transformProps(chartProps: ChartProps) {
   const { width, height, formData, queryData } = chartProps;
-  const { colorScheme, rotation, sizeTo, sizeFrom } = formData;
+  const { colorScheme, rotation, sizeTo } = formData;
 
   return {
     colorScheme,
     data: transformData(queryData.data, formData),
     height,
     rotation,
-    sizeRange: [sizeFrom, sizeTo],
+    sizeRange: [0, sizeTo],
```

Comment:
Why do we need to remove `sizeFrom`?

Reply:
I think size should start from `0`

Reply:
if it starts from '0', does it mean for the least frequent word, its font size will be 0?  We may need to set a better lower bound.  Also, does it mean we want to also deprecate it from wordcloud controls?

Reply:
domain will be `[0, max-frequency]`. Yes, should deprecate the control